packages/terra-clinical-action-header/lib/ActionHeader.js

```diff
@@ -0,0 +1,243 @@
+'use strict';
```

Comment:
The `lib` directory is now [ignored from version control](https://github.com/cerner/terra-clinical/blob/master/.gitignore#L53) and [generated via the compile script after running npm install and when publishing an npm package](https://github.com/cerner/terra-clinical/blob/master/package.json#L55-L56).

This means we can delete this file.